ccloud/beginner-cloud/start.sh

```diff
@@ -89,10 +89,12 @@ ccloud environment use $ENVIRONMENT
 # Create a new Kafka cluster and specify it as the default
 ##################################################
 
-CLUSTER_NAME="demo-kafka-cluster"
+CLUSTER_NAME="${CLUSTER_NAME:-demo-kafka-cluster}"
+CLUSTER_CLOUD="${CLUSTER_CLOUD:-gcp}"
+CLUSTER_REGION="${CLUSTER_REGION:-us-central1}"
 echo -e "\n# Create and specify active Kafka cluster"
-echo "ccloud kafka cluster create $CLUSTER_NAME --cloud gcp --region us-central1"
-OUTPUT=$(ccloud kafka cluster create $CLUSTER_NAME --cloud gcp --region us-central1)
+echo "ccloud kafka cluster create $CLUSTER_NAME --cloud $CLUSTER_CLOUD --region $CLUSTER_REGION"
```

Comment:
Should this change be in this PR?  It will make pint merge tricky (this should propagate, other changes should not)

Reply:
great point, i'll re-target this bit

Reply:
that part is reverted and retargeted for #578